diff --git a/src/subshell.c b/src/subshell.c
index ebd1667..4ac3601 100644
--- a/src/subshell.c
+++ b/src/subshell.c
@@ -128,7 +128,8 @@ static int subshell_pty_slave = -1;
 
 /* The key for switching back to MC from the subshell */
 /* *INDENT-OFF* */
-static const char subshell_switch_key = XCTRL ('o') & 255;
+//static const char subshell_switch_key = XCTRL ('o') & 255;
+static const char subshell_switch_key = ALT ('z') & 255;
 /* *INDENT-ON* */
 
 /* For reading/writing on the subshell's pty */
